src/clj_statsd.clj

```diff
@@ -43,7 +43,7 @@
 (defn send-stat
   "Send a raw metric over the network."
   [prefix content tags]
-  (let [fmt (format-stat prefix content tags)]
+  (let [fmt ^String (format-stat prefix content tags)]
```

Comment:
A type hint on the `format-stat` defn would yield better results

Reply:
SGTM. Done.